recipes/scripttest/bld.bat

```diff
@@ -0,0 +1,3 @@
+pytest
```

Comment:
Is there a reason why you are running `pytest` in the build script before anything is installed?  Typically any tests are run on the built package using the `test` section of the yaml file (although typically only "existence" import tests are run). If it is not necessary to run pytest, please delete the `bld.bat` and `build.sh` and use the `script` section of `build` as you did originally.